index.html

```diff
@@ -1,62 +1,246 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="styles.css">
-  <script src="https://kit.fontawesome.com/b957cbbe16.js" crossorigin="anonymous"></script>
-
-  <title>Portfolio</title>
-</head>
-<body>
-  <header class="mobile-container">
-    <img src="images/Light.svg" alt="light svg">
-    <nav>
-    <div> 
-      <a class="viewmobile">Welcome
-        <i class="fa-solid fa-bars"></i>
-      </a>
-     
-    </div>
-     
-    </nav>
-
-
-  </header>
-  
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b957cbbe16.js" crossorigin="anonymous"></script>
+
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <header class="mobile-container">
+      <img src="images/Light.svg" alt="light svg">
+      <nav>
+        <div>
+          <a class="viewmobile">Welcome
+            <i class="fa-solid fa-bars"></i>
+          </a>
+
+        </div>
+
+      </nav>
+
+
```

Comment:
In order to have a nice and clean repo, is not recommended to have so many empty lines. :sunglasses: 

- [ ] Please remove empty lines from your HTML file. You can keep if you want some empty lines, but only as separators between big sections such as  `head` and `body`, or  `header` and `main`.